decidim-surveys/app/views/decidim/surveys/surveys/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<% add_decidim_meta_tags(
+  description: translated_attribute(current_component.participatory_space.try(:description)),
+  title: translated_attribute(current_component.name),
+  url: surveys_url,
+  resource: current_component
+) %>
+
+<% add_decidim_page_title(t("decidim.surveys.directory.surveys.index.surveys")) %>
+
+<% content_for :aside do %>
+  <% title = current_component.present? ? translated_attribute(current_component.name) : t("decidim.components.surveys.name") %>
+  <h1 class="title-decorator"><%= title %></h1>
+
+  <%= render layout: "decidim/shared/filters", locals: { filter_sections:, skip_to_id: "surveys" } do %>
+  <% end %>
+<% end %>
+
+<%= render layout: "layouts/decidim/shared/layout_two_col" do %>
+    <%= render partial: "decidim/shared/component_announcement" %>
```

Comment:
Is here the only place where the announcement is shown?
This extracts the announcement from the component settings, where is the announcement from the survey settings shown? Maybe we should add it in the "show" view if there's any.

Reply:
There is no show view.

Reply:
ok for the "component's settings" announcement but, for this one:
![imatge](https://github.com/user-attachments/assets/9cddc53f-6e3a-4add-bf3b-f3e8668fd147)
Where it appears?

Reply:
It appears in the show view now, thanks.

Reply:
I added a test for it.